assets/app/scripts/directives/statusIcon.js

```diff
@@ -7,7 +7,11 @@ angular.module('openshiftConsole')
         restrict: 'E',
         templateUrl: 'views/directives/_status-icon.html',
         scope: {
-          status: '='
+          status: '=',
+          disableanimation: "="
```

Comment:
I'm not a fan of multi word variables that aren't camelCased (and then hyphenated in the template)